src/main/java/io/vertx/junit5/VertxTestContext.java

```diff
@@ -238,6 +246,7 @@ public synchronized Checkpoint checkpoint(int requiredNumberOfPasses) {
    *
    * @param <T> the asynchronous result type.
    * @return the handler.
+   * @see #failingThenComplete()
    */
   public <T> Handler<AsyncResult<T>> completing() {
```

Comment:
Uhm i get your point, how you feel about naming this as `succedingThenComplete` to be consistent with `failingThenComplete`? Also @jponge

Reply:
Yes `succedingThenComplete` would make sense alongside a deprecated `succeeding`